### PR TITLE
Update Set up API Gateway Docker scripts section

### PR DESCRIPTION
**Pull request by andreamussap**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by andreamussap** [comment]
@alexearnshaw this is one of the issues that I spot while reading the docs, it's a pain for a user to find this package. Can you please review?

https://deploy-preview-221--axway-open-docs.netlify.com/docs/apim_installation/apigw_containers/docker_scripts_prereqs/

File: content/en/docs/apim_installation/apigw_containers/docker_scripts_prereqs.md [hunk 1]
```diff
@@ -78,7 +79,7 @@ Run the `docker network` command:
 docker network create api-gateway-domain
 ```
 
-This example creates a Docker network called `api-gateway-domain`. For more details on the `docker network` command, see the [Docker user documentation](https://docs.docker.com/ "https://docs.docker.com/").
+This example creates a Docker network called `api-gateway-domain`. For more details on the `docker network` command, see the [Docker user documentation](https://docs.docker.com/ "https\://docs.docker.com/").
```
Comment (alexearnshaw) [inline content/en/docs/apim_installation/apigw_containers/docker_scripts_prereqs.md]: an extra backslash has been added to the URL here?

Comment (andreamussap) [reply content/en/docs/apim_installation/apigw_containers/docker_scripts_prereqs.md]: Eagle eye! How did you spot this if even with the odd link it was working anyways!? :) I don't know what happened there but it's fixed now. Thanks

Comment (alexearnshaw) [reply content/en/docs/apim_installation/apigw_containers/docker_scripts_prereqs.md]: It was highlighted on the diff,  I always check the diff first before I check the preview :)

**Pull request merged**